turbo.json

```diff
@@ -130,11 +130,13 @@
     },
     "storybook:build":  {
       "cache": true,
-      "dependsOn": []
+      "dependsOn": [],
+      "outputs": ["storybook-static"]
```

Comment:
is there any way to catch issues like this before merging, in the future?

Reply:
PR Preview deployments to S3 😉

Reply:
that'll do it!